### PR TITLE
GCW-3278-Install FaceBook full SDK

### PR DESCRIPTION
**Pull request by abulsayyad123**
### Ticket Link: 

https://jira.crossroads.org.hk/browse/GCW-3278

### What does this PR do?

This PR 
- Adds Facebook SDK for mobile app 
- Restrict facebooksdk.js loading for mobile apps.

Pease review the PR

### CODE REVIEW

**Review by bharat619 — APPROVED** [review]

File: cordova/config.xml [hunk 1]
```diff
@@ -108,4 +108,8 @@
   <plugin name="cordova-plugin-globalization" spec="https://github.com/apache/cordova-plugin-globalization.git"/>
   <plugin name="cordova-plugin-inappbrowser" spec="https://github.com/apache/cordova-plugin-inappbrowser.git"/>
   <plugin name="cordova-plugin-dialogs" spec="https://github.com/apache/cordova-plugin-dialogs.git"/>
+  <plugin name="cordova-plugin-twiliovoicesdk" spec="^6.4.0">
+    <variable name="APP_ID" value="315911462935779" />
+    <variable name="APP_NAME" value="goodcity" />
+  </plugin>
```
Comment (patrixr) [inline cordova/config.xml]: Is this related to facebook ? 🤔

**Review by patrixr — APPROVED** [review]

**Review by shreyas098 — APPROVED** [review]

File: cordova/config.xml (hunk 1 above)
Comment (abulsayyad123) [reply cordova/config.xml]: it was by mistake :D
